src/02-common-principles.md

```diff
@@ -73,6 +76,13 @@ misunderstanding we clarify them here.
     acquisition parameters and task (however events can change from run to run
     due to different subject response or randomized nature of the stimuli). Run
     is a synonym of a data acquisition.
+    Note that "uninterrupted" may look different by modality due to the nature of the
+    recording.
+    For example, in [MRI](04-modality-specific-files/01-magnetic-resonance-imaging-data.md),
+    if a subject leaves the scanner, the acquisition must be restarted.
+    In [PET](04-modality-specific-files/09-positron-emission-tomography.md),
+    the start of acquisition is at injection, so a subject may leave and re-enter the
```

Comment:
Not strictly correct wording, as the injection may start at time X, but the participant will not necessarily be entering the scanner (i.e. acquisition start) until time Y.

```suggestion
    for some types of acquisition (dynamic PET), the subject may leave and re-enter the scanner without interrupting the scan. 
```

https://link.springer.com/chapter/10.1007%2F3-540-48714-X_8

Reply:
For simplicity went with:

> For some types of [PET](04-modality-specific-files/09-positron-emission-tomography.md),
> a subject may leave and re-enter the scanner without interrupting the scan.

If additional text would be useful here, please let me know.

Reply:
Maybe I would suggest adding the term acquisition, so: For some types of PET acquisitions, a subject may leave and re-enter the scanner without interrupting the scan.